src/examiner/examiner.js

```diff
@@ -18,21 +18,20 @@
  * @param {!Window} win
  */
 function detectLongTasks(win) {
-  const observer = new win.PerformanceObserver(function (entryList) {
+  const observer = new win.PerformanceObserver((entryList) => {
+    /** @type {!Array<!PerformanceLongTaskTiming>} */
```

Comment:
The observer is only observing `'long-tasks'`. This type-cast allows access to the `attribution` property, which is not present on all `PerformanceEntry`s